### DISCUSSION TITLE
cursor is not working <Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by scoobydube21616** [community/community#78840] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

ok i am new to the forum i have been trying to figure out myall of a sudden my cursor does not follow my work like it has in the past i have got the new firmware an correct drivers all up to date but still it does not move with the picture i am cutting ir engraving

**Comment by ghostinhershell** [comment]
Hi @scoobydube21616,

Thanks for joining us in the GitHub Community, we're glad you're here! We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic. It sounds like reaching out to a tech support professional would be the best direction for you to go for this issue.

Best of luck!
